lib/jobs/ipmi-catalog.js

```diff
@@ -110,11 +110,26 @@ function ipmiCatalogJobFactory(
         assert.string(command);
         assert.object(obmSettings);
 
-        return [ 
+        // Some commands end with " || true"
+        // This tail needs to be deleted from the command args
+        // Otherwise the tail passed to ipmitool child process will cause error response
+        // this tail is stored and used to determine whether continue sending other commands
+        var ignore = false;
+        var tailIdx = command.indexOf(' ||');
```

Comment:
Joel added "|| true" before in compute and mgmt node discovery, as well as command parser (https://github.com/RackHD/on-tasks/pull/35, https://github.com/RackHD/on-tasks/pull/60). The aim is to not block commands after the failure one and collect other catalog information. Although the failure case is rare when using child process in this job, I still would like to cover it and use this flag to catch error and continue processing (https://github.com/iceiilin/on-tasks/blob/mgmt_srv/lib/jobs/ipmi-catalog.js#L160). Also I don't plan to change command parser that is used commonly at this time (since the same PR will be open to 1.1.0 branch too)